dashboard/app/views/home/_homepage.html.haml

```diff
@@ -14,6 +14,51 @@
 
 .clear{style: "clear: both"}
 
+- if current_user
+  - if current_user.teacher?
+    #terms_reminder
+      - unless current_user.accepted_latest_terms?
+        = render template: 'api/accept_terms_reminder'
+
+#terms-modal.modal{'data-backdrop' => 'static', 'data-keyboard' => 'false', style: 'display: none;'}
```

Comment:
Having all of this modal code in here seems redundant, but that's how I could get it to work. Is there are better way to reference this without having it on this page?